Extension/src/LanguageServer/ui.ts

```diff
@@ -344,6 +344,34 @@ export class UI {
         return (selection) ? selection.index : -1;
     }
 
+    public async showSelectCompiler(paths: string[]): Promise<number> {
+        const options: vscode.QuickPickOptions = {};
+        options.placeHolder = localize("select.compile.commands", "Select a compiler to configure for IntelliSense");
+
+        const items: IndexableQuickPickItem[] = [];
+        for (let i: number = 0; i < paths.length; i++) {
+            let option: string | undefined;
+            let isCompiler: boolean = false;
+            if (paths[i].indexOf("\\") > 0) {
+                if (paths[i].split("\\").pop() !== undefined) {
+                    option = paths[i].split("\\").pop();
+                    isCompiler = true;
+                }
+            } else {
+                isCompiler = false;
+            }
+            if (option !== undefined && isCompiler) {
+                const path: string | undefined = paths[i].replace(option, "");
+                items.push({ label: option, description: localize("found.string", "Found at") + path, index: i });
```

Comment:
This results in `Found at/<path>`, etc. It should be something like `localize("found.string", "Found at {0}", path)`